docs/rfcs/0004-ecs-deployment.md

```diff
@@ -0,0 +1,62 @@
+- Start Date: (2021-03-02)
+- Target Version: (1.x / 2.x)
```

Comment:
@nghialv 
What target version should I specify?

Reply:
We have not decided the exact version number for this yet.
But let's set it by `1.1.0`.